Sources/Tracing/SpanProtocol.swift

```diff
@@ -327,7 +327,7 @@ public enum SpanAttribute: Equatable {
     case string(String)
     case stringArray([String])
 
-    case __DO_NOT_SWITCH_EXHAUSTIVELY_OVER_THIS_ENUM
+    case __DO_NOT_SWITCH_EXHAUSTIVELY_OVER_THIS_ENUM_USE_DEFAULT_INSTEAD
```

Comment:
Maybe we can add some written comment to this case as well clarifying this a bit more?

Reply:
We'll add in a follow up

Reply:
We'll add in a follow up